CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributor guidelines
+
+We'd love to accept your patches and contributions to this project. There are just a few small guidelines you need to follow.
+
+When contributing to this project, you must agree that you have authored the content, that you have the necessary rights to the content and that the content you contribute may be provided under the [APACHE LICENSE, VERSION 2.0](LICENSE).
+
+### **What do I need to know to help?**
+
+If you are looking to help to with a code contribution, our project uses Go Lang. If you don't feel ready to make a code contribution yet, no problem! You can also check out the documentation <https://github.com/score-spec/docs>.
+
+If you are interested in making a code contribution and would like to learn more about the technologies that we use, check out the list below.
+
+- [https://humanitec.com](https://humanitec.com/)
```

Comment:
Should we link to the docs here as well? https://docs.humanitec.com/introduction/readme